ci-operator/config/rh-ecosystem-edge/kernel-module-management/rh-ecosystem-edge-kernel-module-management-release-1.1.yaml

```diff
@@ -0,0 +1,180 @@
+base_images:
+  go-builder:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.18-openshift-4.12
+  golangci-lint:
+    name: golangci-lint
+    namespace: ci
+    tag: v1.48.0
+  operator-sdk:
+    name: "4.13"
+    namespace: origin
+    tag: operator-sdk
+binary_build_commands: make manager
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+images:
+- build_args:
+  - name: TARGET
+    value: manager
+  dockerfile_path: Dockerfile
+  to: kernel-module-management-operator
+- build_args:
+  - name: TARGET
+    value: manager-hub
+  dockerfile_path: Dockerfile
+  to: kernel-module-management-operator-hub
+- dockerfile_path: Dockerfile.signimage
+  inputs:
+    ubi8-minimal-entitled:
+      as:
+      - registry.access.redhat.com/ubi8/ubi-minimal:8.7
+  to: kernel-module-management-signimage
+- dockerfile_path: Dockerfile.must-gather
+  to: kernel-module-management-must-gather
+- dockerfile_literal: |
+    FROM ubi8-minimal
+    RUN rm -rf /etc/yum.repos.d/*
+    RUN curl http://base-4-12-rhel8.ocp.svc > /etc/yum.repos.d/base-4-12-rhel8.repo
+  to: ubi8-minimal-entitled
+operator:
+  bundles:
+  - as: operator-bundle
+    dockerfile_path: bundle.Dockerfile
+    skip_building_index: true
+  - as: hub-operator-bundle
+    dockerfile_path: bundle-hub.Dockerfile
+    skip_building_index: true
+  substitutions:
+  - pullspec: quay.io/edge-infrastructure/kernel-module-management-operator:latest
+    with: pipeline:kernel-module-management-operator
+  - pullspec: quay.io/edge-infrastructure/kernel-module-management-operator-hub:latest
+    with: pipeline:kernel-module-management-operator-hub
+  - pullspec: quay.io/edge-infrastructure/kernel-module-management-signimage:latest
+    with: pipeline:kernel-module-management-signimage
+promotion:
+  excluded_images:
+  - ubi8-minimal-entitled
+  namespace: edge-infrastructure
+  tag: release-1.1
+  tag_by_commit: true
+releases:
+  latest:
+    release:
+      channel: fast
+      version: "4.12"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: unit-tests-pr
+  steps:
+    test:
+    - as: run-and-upload-codecov
+      commands: |
+        ci/prow/unit-tests
+        curl -Os https://uploader.codecov.io/latest/linux/codecov
+        chmod +x codecov
+        ./codecov -t "$(cat /var/run/secrets/codecov-token)" -r "${REPO_OWNER}/${REPO_NAME}" -C "${PULL_PULL_SHA}" -P "${PULL_NUMBER}" -B "${PULL_HEAD_REF}" -N "${PULL_BASE_SHA}"
+      credentials:
+      - mount_path: /var/run/secrets
+        name: codecov-token
+        namespace: test-credentials
+      from: src
+      resources:
+        requests:
+          memory: 1Gi
+- as: unit-tests-main
```

Comment:
Why are we testing `main` in the `release-1.1` branch?

Reply:
- before merging: to get the coverage in the current branch;
- after merging: to get the coverage on the base branch, so that for the next PR we can compare coverage between the base branch and the pull request's.